Sources/Private/Model/DotLottie/DotLottieManifest.swift

```diff
@@ -10,21 +10,10 @@ import Foundation
 /// Manifest model for .lottie File
 struct DotLottieManifest: Codable {
 
-  // MARK: Lifecycle
-
-  init(animations: [DotLottieAnimation], version: String, author: String, generator: String) {
-    self.animations = animations
-    self.version = version
-    self.author = author
-    self.generator = generator
-  }
-
-  // MARK: Internal
-
   var animations: [DotLottieAnimation]
-  var version: String
-  var author: String
-  var generator: String
+  var version: String?
+  var author: String?
+  var generator: String?
```

Comment:
The animation in #1945 was also failing to load because the manifest didn't provide these properties. Seems fine to make them optional, especially since we don't use these anywhere.